src/mumble/VolumeSliderWidgetAction.h

```diff
@@ -0,0 +1,33 @@
+// Copyright 2005-2020 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#ifndef MUMBLE_MUMBLE_VOLUMEWIDGETACTION_H_
+#define MUMBLE_MUMBLE_VOLUMEWIDGETACTION_H_
+
+
+#include <QMap>
+
+#include "ClientUser.h"
+
+#include <QtWidgets/QSlider>
+#include <QtWidgets/QWidgetAction>
+#include <QtWidgets/qtooltip.h>
+
+class VolumeSliderWidgetAction : public QWidgetAction {
+	Q_OBJECT
+
+
+public:
+	VolumeSliderWidgetAction(QObject *parent = nullptr);
+
+
+protected:
+	QSlider *qsVolumeSliderWidgetAction;
+	void setDBSliderValue(float value);
```

Comment:
Why is there a "DB" in the name?

Reply:
the DB was from decibel, it should have been dB